spk/salt-minion/Makefile

```diff
@@ -0,0 +1,52 @@
+SPK_NAME = salt-minion
+SPK_VERS = 3001.1
+SPK_REV = 1
+SPK_ICON = src/salt-minion.png
+
+PIP = $(WORK_DIR)/../../../native/python3/work-native/install/usr/local/bin/pip
+BUILD_DEPENDS = cross/python3
+SPK_DEPENDS = "python3>=3.7"
+
+# Requirements file generation
+# /usr/local/python3/bin/python3 -mvirtualenv --python=python3  salt-env
+# virtualenv --python=python3  salt-env
+# source salt-env/bin/activate
+# pip install salt
+# pip freeze > requirements.txt
+WHEELS = src/requirements.txt
+
+MAINTAINER = SynoCommunity
+DESCRIPTION = Salt, a new approach to infrastructure management, is easy enough to get running in minutes, scalable enough to manage tens of thousands of servers, and fast enough to communicate with those servers in seconds.
+RELOAD_UI = yes
+DISPLAY_NAME = Salt Minion
+CHANGELOG = "Update to 3001.1"
+
+HOMEPAGE   = https://www.saltstack.com/
+LICENSE    = Apache
+
+POST_STRIP_TARGET = salt-minion_extra_install
+
+# Service configuration
+CONF_DIR = src/conf
+SERVICE_USER = auto
+SERVICE_SETUP = src/service-setup.sh
+STARTABLE = yes
+
+# Pure Python package, make sure ARCH is not defined
+#override ARCH=
+
```

Comment:
In fact I am wrong, without busybox, `ARCH` must be reset as empty to create an non-arch package.

Reply:
Probably I am still wrong, if at least one Python module from requirements needs native built (so it is not "any"), package cannot be "non arch". Please check and confirm.

Reply:
I commented it out because some of the python dependencies (included as wheel packages) contain arch specific binary libraries:
$ ls spk/salt-minion/work-rtd1296-6.1/wheelhouse
certifi-2020.6.20-py2.py3-none-any.whl         pycryptodomex-3.9.8-cp37-cp37m-linux_aarch64.whl
chardet-3.0.4-py2.py3-none-any.whl             PyYAML-5.3.1-cp37-cp37m-linux_aarch64.whl
distro-1.5.0-py2.py3-none-any.whl              pyzmq-19.0.2-cp37-cp37m-linux_aarch64.whl
idna-2.10-py2.py3-none-any.whl                 requests-2.24.0-py2.py3-none-any.whl
Jinja2-2.11.2-py2.py3-none-any.whl             requirements.txt
MarkupSafe-1.1.1-cp37-cp37m-linux_aarch64.whl  salt-3001.1-py3-none-any.whl
msgpack-1.0.0-cp37-cp37m-linux_aarch64.whl     urllib3-1.25.10-py2.py3-none-any.whl

Reply:
done